html/YANPIWS.js

```diff
@@ -29,20 +29,21 @@ function loadXMLDoc(URL, targetId, callback) {
 }
 
 /**
- * handle resizing clock to big then small
+ * handle resizing clock to big then small 
+ * by toggling between big_time and small_time classes
  */
-// todo - use class for defaultSize, remove from signature
 function setClockSize(state, defaultSize){
     if (state == 'big'){
-        // todo - put all this in a class and then just add and remove class
-        $('#time').css('font-size', '127pt').css('text-align', 'center').css('width', '100%');
-        $('#date').css('text-align', 'center').css('width', '100%');
-        $('.big_clock_hide').hide();
+        var small_times = $('.small_time')
+        small_times.addClass("big_time")
+        small_times.removeClass("small_time")
+
+        console.log('using big size');
     } else {
-        // todo - put all this in a class and then just add and remove class
-        $('#time').css('font-size', defaultSize + 'pt').css('text-align', 'left').css('width', 'inherit');
-        $('#date').css('text-align', 'left').css('width', 'inherit');
-        $('.big_clock_hide').show();
+        var big_times = $('.big_time')
+        big_times.addClass("small_time")
+        big_times.removeClass("big_time")
+
         console.log('using small size: ' + defaultSize);
```

Comment:
```suggestion
        $('.big_time').addClass("small_time")
        $('.big_time').removeClass("big_time")
```
Less code, less debug ;)